internal/utils/logger.go

```diff
@@ -39,15 +39,6 @@ func parseLogLevel(logLevel int) slog.Level {
 // TODO(agamez): use it as a dependency,
 // once available at https://pkg.go.dev/github.com/trento-project/agent@v0.0.0-20250417081934-5aa03367504a/pkg/utils
```

Comment:
https://pkg.go.dev/github.com/trento-project/agent still not showing the latest version, unfortunately.

Reply:
If not already done we might track this in the backlog